lib/kramdown/converter/prismic.rb

```diff
@@ -173,14 +173,14 @@ def convert_sub_html_element(element, type)
       def convert_html_element(element)
         if element.value == 'iframe'
           {
+            type: 'embed',
+            data: {
+              embed_url: element.attr['src'],
+              type: 'video'
```

Comment:
Unfortunately, we don't really know that it's an embed. But when I tested it on primisc, it was working as expected.

Reply:
If we don't choose video, the embed import fails completely. What are your thoughts to resolve this